app/views/assistants/_assistant.html.erb

```diff
@@ -8,6 +8,7 @@
               flex justify-between items-center
               mb-1 p-1 pl-2 pr-2 mr-5
             hover:bg-gray-100 dark:hover:bg-gray-700
+            bg-gray-50 dark:bg-gray-900
```

Comment:
@florianfelsing One small thing I missed in your design change: content scrolls beneath the very top sticky assistant. This was an easy fix to add the background back:
<img width="369" alt="image" src="https://github.com/the-dot-bot/hostedgpt/assets/35061/7621aec3-933a-47c5-a7c6-06e9a637ce0f">